vital/types/detected_object.h

```diff
@@ -92,6 +103,17 @@ class VITAL_EXPORT detected_object
                    double confidence = 1.0,
                    detected_object_type_sptr classifications = detected_object_type_sptr() );
 
+  /**
+   * @brief Create detected object with a geo_point and other attributes.
+   *
+   * @param geo_pt Geographic location of the detection, in world coordinates.
+   * @param confidence Detectors confidence in this detection.
+   * @param classifications Optional object classification.
+   */
+  detected_object(const geo_point& geo_pt,
```

Comment:
Style again... also, eww, please use `= {}` or `= nullptr` (yes, I know above is doing the same... feel free to fix it also!)